src/components/Graphs/Graphs.tsx

```
@@ -0,0 +1,111 @@
+    pv: 3800,
+    amt: 2500,
+  },
+  {
+    name: "Page G",
+    uv: 3490,
+    pv: 4300,
+    amt: 2100,
+  },
+];
+
+const CustomizedLabel: FunctionComponent<any> = (props: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)